wandb/integration/tensorboard/log.py

```diff
@@ -320,22 +330,24 @@ def _log(
             or timestamp - STEPS["global"]["last_log"] >= RATE_LIMIT_SECONDS
         ):
             if history is None:
-                wandb.run._log({})
+                if wandb.run is not None:
```

Comment:
probably a better place to check this in the estimator_hook: https://github.com/wandb/client/blob/master/wandb%2Fintegration%2Ftensorflow%2Festimator_hook.py#L48

we can add a warning message that they need to init the run before calling the hook....
like all of our other integrations

Reply:
Changed to raising an error, like everywhere else. 
This should really never happen, it's mainly for mypy since `wandb.run` is `Optional`.